<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>775</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>817</width>
    <height>775</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>0</y>
      <width>891</width>
      <height>801</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Images/5655049.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>150</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>46</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#playButton{
background-color:rgba(255,107,107,255);
color:rgba(255,255,255,200);
border-radius:10px;
}
QPushButton#playButton:pressed{
padding-left:5px;
padding-top:5px;
background-color:rgba(255,84,84,200);
background-position:calc(100%-10px)center;
}
QPushButton#playButton:hover{
background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(255, 107, 107, 255), stop:1 rgba(244, 168, 168, 219));
}
</string>
    </property>
    <property name="text">
     <string>â–¶</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#backButton{
background-color:rgba(255,107,107,255);
color:rgba(255,255,255,200);
border-radius:5px;
}
QPushButton#backButton:pressed{
padding-left:5px;
padding-top:5px;
background-color:rgba(255,84,84,200);
background-position:calc(100%-10px)center;
}
QPushButton#backButton:hover{
background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(255, 107, 107, 255), stop:1 rgba(244, 168, 168, 219));
}
</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="image">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>151</width>
      <height>151</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>151</width>
      <height>151</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 50px solid;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Images/listimg.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="NameList">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border:2px solid rgba(0,0,0,0);
border-bottom-color:rgba(46,82,101,200);
color:rgb(0,0,0);
padding-bottom:7px;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;PLAYLIST&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>170</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border:2px solid rgba(0,0,0,0);
border-bottom-color:rgba(46,82,101,200);
color:rgb(0,0,0);
padding-bottom:7px;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="profile">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>170</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Images/profile.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="MusicLists">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>821</width>
      <height>461</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 164, 164, 255), stop:1 rgba(255,255,255, 255));
color: rgb(255, 255, 255);
selection-color: rgb(51, 0, 25);</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>817</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionAdd_Songs"/>
    <addaction name="actionRemove_Selected_Song"/>
    <addaction name="actionRemove_All_Song"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_Songs">
   <property name="text">
    <string>Add Songs</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionRemove_Selected_Song">
   <property name="text">
    <string>Remove Selected Song</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionRemove_All_Song">
   <property name="text">
    <string>Remove All Song</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
